website/content/maintainers/github/templates/required/governance-elections.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "The Steering Committee Elections Template"
+linkTitle: "Elections Governance"
+date: 2022-07-14
+draft: true
+weight: 32
+---
+
+Stub; TBR.
```

Comment:
I'm unclear on your plans for this document. Since it's flagged with `draft: true` it will not be published to the site, even after it's merged to main. So from that standpoint, I don't care that it's a stub but is that what you intended?

Reply:
Oh, it's not a stub anymore, I forgot to take that out.

Reply:
I was just creating a document so that I could set up the links from governance-intro.  Since we don't know when we can complete these two docs, is it better to have them as stubs in our docs, or to omit them entirely and have the items in governance-intro say (documentation pending)?

Reply:
See changed version.

Reply:
Looks good! 👍